content/en/blog/2025/stabilizing-rpc-conventions.md

```diff
@@ -0,0 +1,98 @@
+---
+title: >-
+  Announcing the RPC Semantic Conventions stabilization project
+linkTitle: Stabilizing RPC Semantic Conventions
+date: 2025-05-28
+author: >-
+  [Liudmila Molkova](https://github.com/lmolkova) (Microsoft), [Trask
+  Stalnaker](https://github.com/trask) (Microsoft)
+sig: Semantic Conventions
+cSpell:ignore: Dubbo Liudmila Molkova
+---
+
+The Semantic Conventions SIG is excited to kick off the RPC stabilization
+effort!
+
+Following the stabilization of the database conventions in May 2025, we're
+continuing our work to stabilize key areas—and RPC is next.
+
+It takes a village to define a solid convention, especially for a space as
+diverse as RPC technologies, which include gRPC, JSON-RPC, Apache Dubbo, and
+many others. If you work on one of these frameworks, use them extensively, or
+are simply interested in learning more, come join us—we’d love your help!
+
+## Towards reliable telemetry conventions
+
+Reliable, well-defined conventions are the runway for richer telemetry
+experiences. When signal and attribute names stay consistent, everyone can spend
+their time building alerts, dashboards, and visualizations - not firefighting
+breaking changes.
+
+Existing experimental conventions have been in use for quite a while, and
+introducing breaking changes in the corresponding instrumentations would be
+disruptive.
+
+We firmly believe that these changes are essential in the long run to deliver
+high-quality instrumentation that produces actionable, useful telemetry.
+
+To ensure a smooth transition, we are following a
+[graceful migration plan](https://github.com/open-telemetry/semantic-conventions/blob/v1.34.0/docs/rpc/rpc-spans.md?plain=1#L26-L50).
+Instrumentation libraries will:
+
+- Ship the new semantic conventions behind an opt-in flag, side-by-side with the
+  existing ones,
+- Maintain both versions of conventions for an extended period,
+- Provide detailed migration guide.
+
+## How does semantic convention stabilization work?
+
+During the stabilization phase, we review existing conventions to ensure they
+offer meaningful insights for most applications using the technology. We check
+that the conventions enable generic instrumentation with reasonable performance
+overhead, while also accounting for privacy, telemetry volume, consistency, and
+correlation with higher-level application and lower-level transport telemetry.
+
+We aim for conventions that are useful, usable, and extensible.
+
+For RPC, we're focusing on the following major areas:
+
+- Essential signals
+
```

Comment:
Put area title in bold, and drop blank line; here and for the other entries:

```suggestion
- **Essential signals**: 
```

This will make the areas pop out, e.g.:

> <img width="711" alt="image" src="https://github.com/user-attachments/assets/e8ddf1ca-1211-4782-a182-d8e4e6874422" />